### PR TITLE
ADX-562 Added GH actions unit tests

### PR DESCRIPTION
**Pull request by fulior**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install requirements
+      run: |
+        pip install -r requirements.txt
+        pip install -r dev-requirements.txt
+        pip install -e "git+https://github.com/fjelltopp/ckanext-scheming@development#egg=ckanext-scheming"
+        pip install -r https://raw.githubusercontent.com/fjelltopp/ckanext-scheming/development/requirements.txt
+
+        pip install ckantoolkit
```
Comment (tomeksabala) [inline .github/workflows/test.yml:53]: This should already be installed in validation requirements.txt
```
ckantoolkit>=0.0.3
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        pip install -e .
```
Comment (tomeksabala) [inline .github/workflows/test.yml:54]: `-e` is editable mode useful in development. Here we can safetly (and possibly quicker) user simple `pip install <githubURL>` or `pip install .`

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install requirements
+        run: pip install flake8 pycodestyle
+      - name: Check syntax
+        run: |
+          flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
```
Comment (tomeksabala) [inline .github/workflows/test.yml:15]: Remove as `flake8 . --count --max-line-length=127 --show-source --statistics` will be enough.

**Review by tomeksabala — CHANGES_REQUESTED** [review]
Some changes required. They should probably be also applied to other PRs with github actions.

File: .github/workflows/test.yml (hunk 3 above)
Comment (fulior) [reply .github/workflows/test.yml]: done

File: .github/workflows/test.yml (hunk 2 above)
Comment (fulior) [reply .github/workflows/test.yml:51]: done, pointed pip install to development archive url (faster than a clone operation)

File: .github/workflows/test.yml (hunk 1 above)
Comment (fulior) [reply .github/workflows/test.yml]: right, didn't notice that :)

**Review by tomeksabala — APPROVED** [review]

**Pull request merged**